sdk/synapse/Azure.Analytics.Synapse.Artifacts/README.md

```diff
@@ -70,7 +70,12 @@ Notebook notebook = new Notebook(
     nbformatMinor: 2,
     new List<NotebookCell>()
 );
-var createdNotebook = notebookClient.StartCreateOrUpdateNotebook("MyNotebook", new NotebookResource(notebook));
+var operation = notebookClient.StartCreateOrUpdateNotebook("MyNotebook", new NotebookResource(notebook));
+while (!operation.HasValue)
+{
+    operation.UpdateStatus();
+}
```

Comment:
@pakrym - why don't we have a sync overload of `WaitForCompletion()` again?  Should our sample code throw in a delay here?

Reply:
Something about blocking threads synchronously for a long time and it being an easy way to introduce thread pool starvation.

Reply:
Might be good to add a sample to https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/LongRunningOperations.md that explains the best practices and trade-offs for waiting on one of these in a sync method then.  Filed https://github.com/Azure/azure-sdk-for-net/issues/14757 to track.